extensions/http-curl/CMakeLists.txt

```
@@ -17,6 +17,12 @@
 # under the License.
 #
 
+if (NOT ((DISABLE_CURL STREQUAL "OFF" OR NOT DISABLE_CURL) AND NOT DISABLE_CIVET))
```

```suggestion
if (NOT ((NOT DISABLE_CURL OR NOT DISABLE_CURL) AND NOT DISABLE_CIVET))
```

This should be about the same, expect for accepting "no" and 0 as well.

---

changed it